docs/Common/Enums/index.md

```diff
@@ -841,6 +843,7 @@
 |VEHICLEDATA_FUELRANGE|27||
 |VEHICLEDATA_ENGINEOILLIFE|28||
 |VEHICLEDATA_ELECTRONICPARKBRAKESTATUS|29||
+|VEHICLEDATA_CLOUDAPPVEHICLEID|30||
```

Comment:
GetVehicleData and OnVehicleData should be updated with some detailed information about this parameter. 

See information from proposal. https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0158-cloud-app-transport-adapter.md#cloudappvehicleid

Also include note that the user might want to reset this field (vehicle change of ownership) and the HMI will have to update this field.

Reply:
Added a description to the enum. Also added this section in the GetVehicleData file 
!!! NOTE
  #### CloudAppVehicleID  
    a) Parameter used by cloud apps or the policy server to identify the head unit  
    b) OEM may choose to reset this value in case the vehicle changes owners
!!!
What exactly does `HMI will have to update this field` refer to. Is this in the case that the cloudAppVehicleID is reset?

Reply:
Yes, I was referring to if cloudAppVehicleID is reset.